.idea/codeStyles/Project.xml

```diff
@@ -9,9 +9,6 @@
         <option name="TAB_SIZE" value="2" />
       </value>
     </option>
-    <AndroidXmlCodeStyleSettings>
-      <option name="USE_CUSTOM_SETTINGS" value="true" />
-    </AndroidXmlCodeStyleSettings>
```

Comment:
Can you revert changes to the codestyle?

Reply:
Done